docs/data/data-grid/columns/columns.md

```diff
@@ -205,13 +206,17 @@ const columns: GridColDef[] = [
 ];
 ```
 
-{{"demo": "RenderCellGrid.js", "bg": "inline"}}
+{{"demo": "RenderCellGrid.js", "bg": "inline", "defaultCodeOpen": false }}
 
-**Note**: It is recommended to also set a `valueFormatter` providing a representation for the value to be used when [exporting](/components/data-grid/export/#exported-cells) the data.
-
-> ⚠️ When using `renderCell` with object cell values
-> remember to handle [sorting](/components/data-grid/sorting/#custom-comparator).
-> Otherwise, sorting won't work.
+> ⚠️ Using `renderCell`, requires paying attention to the following points.
+>
+> - If the value returned `valueGetter` does not correspond to the column `type`, you should:
+>
+>   - handle [sorting](/components/data-grid/sorting/#custom-comparator) by providing `sortComparator` to the column.
+>   - set a `valueFormatter` providing a representation for the value to be used when [exporting](/components/data-grid/export/#exported-cells) the data.
+>
+> - According to [WAI-ARIA](https://www.w3.org/TR/wai-aria-practices-1.1/#grid), only one of the focusable elements contained by the grid should be included in the page tab sequence.
```

Comment:
Move to the accessibility page? I think we'll have other sections there about #4219

Reply:
Done